conda/libmesh-vtk/build.sh

```diff
@@ -19,15 +19,15 @@ cmake .. -G "Ninja" \
     -DCMAKE_PREFIX_PATH:PATH=${VTK_PREFIX} \
     -DCMAKE_INSTALL_PREFIX:PATH=${VTK_PREFIX} \
     -DCMAKE_INSTALL_RPATH:PATH=${VTK_PREFIX}/lib \
```

Comment:
Nice find. These can be hidden unless using `ccmake`. What a pain, thanks for figuring this out.

Reply:
Based on our slack discussion, it does look like this isn't being honored properly....I'll double-check to make sure this hasn't changed as well.

Reply:
Here it is.....this *is* being honored, but the format for the runtime lib has changed. From the build output on rod:
```
Set runtime path of "$PREFIX/libmesh-vtk/lib64/libvtkCommonCore-9.1.so.9.1.0" to "$PREFIX/libmesh-vtk/lib"
```

Reply:
To resolve this, based on our Slack discussion, I added a symlink during the build process, so that `lib` points to `lib64`.